src/layer/AbstractLayer.ts

```diff
@@ -356,6 +360,12 @@ export class AbstractLayer {
               this.roundCoordinates(currentFlyoverGeometry),
               this.roundCoordinates(tiles[tileIndex].geometry.coordinates as Geom),
             );
+
+            // update the lowest cloud cover:
+            if (tiles[tileIndex].meta.cloudCoverPercent < tiles[tileIndex - 1].meta.cloudCoverPercent) {
```

Comment:
I think this check would be clearer if the comparison would be done against the saved value and not the value of the previous element 

```js
if (tiles[tileIndex].meta.cloudCoverPercent < flyovers[flyoverIndex].meta.lowestCloudCoverPercent) {
  ...
}
```